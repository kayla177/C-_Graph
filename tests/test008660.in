ENTITY MLMROB6G name_MLMR paper
ENTITY PSN91B3E name_PSN9 paper
ENTITY GENGD3NZ name_GENG author
ENTITY W9M1P7WZ name_W9M1 paper
ENTITY BAPWNNUE name_BAPW conference
ENTITY VK8IF0N9 name_VK8I institution
ENTITY OSGYV00E name_OSGY author
ENTITY R62Z3FHR name_R62Z paper
ENTITY UPFE22FU name_UPFE author
ENTITY OARAXSVJ name_OARA institution
RELATIONSHIP BAPWNNUE published_in OARAXSVJ 8.3
RELATIONSHIP UPFE22FU collaborated_with R62Z3FHR 4.0
RELATIONSHIP PSN91B3E affiliated_with MLMROB6G 4.6
RELATIONSHIP GENGD3NZ cited_by W9M1P7WZ 7.8
RELATIONSHIP VK8IF0N9 published_in GENGD3NZ 7.8
RELATIONSHIP UPFE22FU authored_by VK8IF0N9 4.5
RELATIONSHIP OSGYV00E affiliated_with BAPWNNUE 8.2
RELATIONSHIP BAPWNNUE published_in R62Z3FHR 9.4
RELATIONSHIP GENGD3NZ published_in R62Z3FHR 3.9
RELATIONSHIP R62Z3FHR collaborated_with OARAXSVJ 0.6
RELATIONSHIP W9M1P7WZ authored_by R62Z3FHR 3.8
RELATIONSHIP MLMROB6G affiliated_with OARAXSVJ 0.7
RELATIONSHIP OSGYV00E affiliated_with VK8IF0N9 8.2
RELATIONSHIP BAPWNNUE cited_by W9M1P7WZ 2.5
RELATIONSHIP W9M1P7WZ presented_at MLMROB6G 9.7
HIGHEST
DELETE PSN91B3E
PRINT GENGD3NZ
PATH BAPWNNUE W9M1P7WZ
HIGHEST
PATH OSGYV00E BAPWNNUE
HIGHEST
PRINT OSGYV00E
HIGHEST
PRINT R62Z3FHR
EXIT
