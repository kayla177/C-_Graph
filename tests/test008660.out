success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
PSN91B3E OSGYV00E 38.1
success
W9M1P7WZ VK8IF0N9 R62Z3FHR
BAPWNNUE R62Z3FHR W9M1P7WZ 13.2
MLMROB6G OSGYV00E 33.5
OSGYV00E BAPWNNUE 8.2
MLMROB6G OSGYV00E 33.5
BAPWNNUE VK8IF0N9
MLMROB6G OSGYV00E 33.5
UPFE22FU BAPWNNUE GENGD3NZ OARAXSVJ W9M1P7WZ
